### PR TITLE
add_gitignore

### PR DESCRIPTION
**Pull request by mindw**
It's seriously missed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+syntax: glob
+build/
+*.pyc
+*.egg-info
+dist/
+*.pyd
+.idea/
```
Comment (giampaolo) [inline .gitignore]: What's ".idea"?

Comment (mindw) [inline .gitignore]: PyCharm.

Comment (mindw) [inline .gitignore]: https://www.gitignore.io/api/python,pycharm

**Comment by giampaolo** [comment]
Thanks a lot.

**Pull request merged**